tests/local_mode/test_language.py

```diff
@@ -50,6 +52,7 @@ def test_vector_non_valid_token_is_zero():
 
     doc = nlp("outofvocabularytoken")
     actual = doc[0].vector
+    # changed from numpy to torch zeros tensor
     zeros = np.zeros(actual.shape)
```

Comment:
Please make this a torch tensor instead of a numpy array.

Reply:
ok I will work on it🙂